packages/server/src/lib.rs

```diff
@@ -153,8 +153,8 @@ impl Server {
 		#[cfg(test)]
```

Comment:
This whole block can be removed now - after some churn we now have this handled at the [bottom of this file](https://github.com/tangramdotdev/tangram/blob/2e6cdd4e7ea65f97a073dc6afecf64f83fa50501/packages/server/src/lib.rs#L1238), the value chosen is higher than the value you needed here.

Reply:
Done, thanks!